screens/EditSignal/index.tsx

```diff
@@ -42,21 +60,22 @@ export default function EditSignal() {
       console.error(error.message)
     },
     onSettled: async () => {
-      await fetchMySignal()
       setIsLoading(false)
```

Comment:
Don't you need to Invalidate the query onSettled as well to ensure we have the current data after the mutation has completed ?